Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.Properties.cs

```diff
@@ -129,6 +154,36 @@ public ICommand ItemClickCommand
         }
 
         /// <summary>
+        /// Gets or sets the selected multiple objects in the collection.
+        /// </summary>
+        /// <value>The object that is used to store selected multiple items. Default is an empty IList collection of object.</value>
+        public IList<object> SelectedItems
+        {
+            get { return (IList<object>)GetValue(SelectedItemsProperty); }
+            set { SetValue(SelectedItemsProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether gets or sets whether the items are clickable or not.
+        /// </summary>
+        /// <value>Default is false.</value>
+        public bool IsItemClickEnabled
+        {
+            get { return (bool)GetValue(IsItemClickEnabledProperty); }
+            set { SetValue(IsItemClickEnabledProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the SelectionMode property of ListViewBase. Does not support runtime changes at the moment.
+        /// </summary>
+        /// <value>Default is single which allows users to only pick single item in the collection.</value>
+        public ListViewSelectionMode SelectionMode
```

Comment:
are you sure, I only see this: https://github.com/Microsoft/UWPCommunityToolkit/pull/182/files#diff-64e763d776268be5661a9954709df928R84